.github/scripts/validate_sidecar_resources.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+#
+# Copyright 2024 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script parses release version from Git tag and set the parsed version to
```

Comment:
Inaccurate comment presumably copied from the other script.

Reply:
Fixed. Thanks.